models/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/StoreTransaction.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.kmp.models
+
+/**
+ * Represents an in-app billing purchase.
+ */
+public data class StoreTransaction(
+    /**
+     * Unique (per store) order identifier for the purchased transaction. Always null for Amazon.
+     * Only available for non-restored Google purchases for Google Play.
+     */
+    val transactionId: String?,
+
+    /**
+     * Product IDs purchased.
+     *
+     * Size can be greater than 1 on Android. This indicates that a multi-line purchase occurred,
+     * which RevenueCat does not support.
+     * Only the first productId will be processed by the SDK.
+     */
+    val productIds: List<String>,
+
+    /**
+     * Time the product was purchased, in milliseconds since the epoch.
+     */
+    val purchaseTime: Long,
+) {
+    public companion object
```

Comment:
I added this to be able to add a static method in the `mappings` module... Maybe there is a better way to organize this but I tried to minimize the changes.

Reply:
Yeah, I don't like too much to have these as global functions... But I think it works for now 👍